examples/example_plugin/example_plugin/template_content.py

```diff
@@ -1,6 +1,24 @@
+from django.urls import reverse
+
 from nautobot.extras.plugins import PluginTemplateExtension
 
 
+class DeviceContent(PluginTemplateExtension):
```

Comment:
Please add docstring/comments to make this example as self-explanatory as possible.

Reply:
`#TODO`